pkg/registry/generic/rest/proxy_test.go

```diff
@@ -135,7 +167,7 @@ func TestServeHTTP(t *testing.T) {
 			backendResponse := "<html><head></head><body><a href=\"/test/path\">Hello</a></body></html>"
 			backendHandler := &SimpleBackendHandler{
 				responseBody:   backendResponse,
-				responseHeader: map[string]string{"Content-Type": "text/html"},
+				responseHeader: test.responseHeader,
```

Comment:
Either default it to also set (and check) "Content-Type": "text/html", or add that to every entry in the table... :/

Can't stop testing that behavior.